package.json

```diff
@@ -5,8 +5,8 @@
   "description": "Fastish HTML",
   "main": "index.js",
   "engines": {
-    "node": "^4.7.0",
-    "npm": "^5.0.0"
+    "node": ">=4.8.0",
```

Comment:
I believe you should remove the `engines` section from this file. We specify the `node` version that Travis must use via `.travis.yml`, and Travis' built in version of `yarn` used to work fine as it was.

Reply:
See https://github.com/travis-ci/travis-ci/issues/4653#issuecomment-169035161 for an explanation of why Travis ignores the versions of `node` and `npm` specified in `package.json`.

Reply:
No change, Travis still fails on Yarn